src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,23 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.Arrays;
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final String validNationality = "Ukrainian";
+    private static final int validMinAge = 35;
+    private static final int validMinPeriodsInUkr = 10;
+
+    @Override
+    public boolean test(Candidate candidate) {
+        int periodsInUkraine = Arrays.stream(candidate.getPeriodsInUkr().split("-"))
```

Comment:
```suggestion
        int periodsInUkraine = Arrays.stream(candidate.getPeriodsInUkr().split(SEPARATOR))
```
this is a string literal too, better to extract to constant